modules/dashboard/README.md

```diff
@@ -0,0 +1,32 @@
+# Dashboard
+
+## Purpose
+
+The Dashboard serves as a landing page for all users logging into LORIS.  It provides introductory information and a primary quick-access interface for user workflows, including outstanding tasks and notifications of recent activity/additions in LORIS.  
+
+The Dashboard also provides at-a-glance live visual summaries of the data stored in LORIS, and dynamically plots data breakdowns that can be used in reporting. 
+
+## Intended Users
+
+All LORIS users can view the dashboard.  
+
+## Scope
+
+The Dashboard provides entrypoints to various workflows, but does not provide any tools for directly modifying data in the dashboard.  
+
+The Study Progression chart views at bottom may or may not count `excluded` candidates, and this should be clearly verified and documented/presented for the use of your LORIS users. 
+
+## Permissions
+
+Components of the Dashboard are governed by permissions related to module-specific privileges. For example, the _Document Repository_ notifications panel is displayed if a user has been granted either the `document_repository_view` or `document_repository_delete` permission. Site access privileges are also enforced. 
+
+## Configurations
+
+Many aspects of the Dashboard are configured in the Configuration module, under the Admin menu. 
+
+Under the _Dashboard_ section of the Configuration module, you can edit 2 settings: 
+* _Project Description_ : the blurb that appears at the top of the Dashboard. This text cannot contain HTML characters (such as line breaks), for security reasons.  
```

Comment:
```suggestion
* _Project Description_ : the blurb that appears at the top of the Dashboard. 
```

HTML can be entered here, though it won't actually render. Instead it'll change things to e.g. `&amp;` instead of `&`.

Reply:
Ah, thanks for the clarification